### PR TITLE
include script file saving

### PR DESCRIPTION
**Pull request by bridgeta**
Introduce saving scripts.

Guidance didn't include explicit instructions to save scripts. 
Included basic step by step instructions before main lesson to save a script and continue saving.

### CODE REVIEW

File: 01-intro-to-r.Rmd [hunk 1]
```diff
@@ -130,6 +131,21 @@ weight_kg <- 100
 
 What do you think is the current content of the object `weight_lb`? 126.5 or 220?
 
+### Saving your code
+
+Up to now, your code has been in the console. This is useful for quick queries
+but not so helpful if you want to revist your work for any reason.
+A script can be opened by pressing the <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + 
+<kbd>N</kbd>. 
+It it wise to save your script file immediately. To do this press <kbd>Shift</kbd> +
```
Comment (fmichonneau) [inline 01-intro-to-r.Rmd:140]: which operating system do you use? I thought that Ctrl + S is the usual shortcut to save a file.

File: 01-intro-to-r.Rmd [hunk 2]
```diff
@@ -130,6 +131,21 @@ weight_kg <- 100
 
 What do you think is the current content of the object `weight_lb`? 126.5 or 220?
 
+### Saving your code
+
+Up to now, your code has been in the console. This is useful for quick queries
+but not so helpful if you want to revist your work for any reason.
```
Comment (fmichonneau) [inline 01-intro-to-r.Rmd:137]: revist -> revisit

File: 01-intro-to-r.Rmd [hunk 3]
```diff
@@ -130,6 +131,21 @@ weight_kg <- 100
...
+A script can be opened by pressing the <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + 
+<kbd>N</kbd>. 
+It it wise to save your script file immediately. To do this press <kbd>Shift</kbd> + 
+<kbd>S</kbd>. This will open a dialogue box where you can decide where to save 
+your script file, and what to name it. The .R file extension is added automatically
```
Comment (fmichonneau) [inline 01-intro-to-r.Rmd:142]: .R -> `.R`

File: 01-intro-to-r.Rmd [hunk 4]
```diff
@@ -130,6 +131,21 @@ weight_kg <- 100
...
+and ensures your file will open with R Studio.
```
Comment (fmichonneau) [inline 01-intro-to-r.Rmd:143]: remove space between R and studio

File: 01-intro-to-r.Rmd (hunk 1 above)
Comment (bridgeta) [reply 01-intro-to-r.Rmd:140]: It is there, on the next line down. Would it better to move the CTRl to the next line also?

**Pull request closed without merging**